tools/README.md

```diff
@@ -0,0 +1,61 @@
+# NNCF Tools
+
+## Memory Monitor
+
+Memory Monitor is a tool that can be used to measure python program RAM footprint in time. It supports multiple memory types:
+
+- `MemoryType.RSS`: Resident Set Size is the portion of memory occupied by a process that is held in RAM.
+
+- `MemoryType.RSS_TOP`: The same type of memory as above, but the values are parsed from a table output of the *top* command. Usually, reports the same values as RSS, but with worse resolution.
```

Comment:
> There is any benefits to use RSS_TOP over RSS?

Not that I'm aware of

> May be keep only RSS to reduce code?

I also thought about it, but decided to keep just in case it may be useful in the future. Don't want this logic to be lost somewhere when it comes the moment where it would be helpful

Reply:
You have two ways to read `/proc/[pid]/stat` file.
First by one line by pstutls, second creates subprocesses 10 times (it is not the fastest operation in second, call `top` and requires logic to parse output and catch errors.
I'm not see any reason to keep it.

Reply:
Removed